.github/workflows/verify.yml

```diff
@@ -0,0 +1,113 @@
+name: Verify
```

Comment:
Note: I'm aiming for a semantic copy of the existing travis job, I'm intentionally not wishing to change behavior were I can avoid it. This will allow for running both Github Actions and Travis in parallel to compare wait times/performance/reliability etc.

https://github.com/rapid7/metasploit-framework/blob/8aaae387ea67c908c71472b3a239078fb3675ac5/.travis.yml